NPE in HyperionScreenService

From the play console:
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:3626)
  at android.app.ActivityThread.-wrap20 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1717)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6710)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:240)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:770)
Caused by: java.lang.NullPointerException: 
  at com.abrenoch.hyperiongrabber.common.HyperionScreenService.onStartCommand (HyperionScreenService.java:162)
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:3609)
```

Seems to come from `final String action = intent.getAction();` when the `intent` is `null`

--- a/common/src/main/java/com/abrenoch/hyperiongrabber/common/HyperionScreenService.java
+++ b/common/src/main/java/com/abrenoch/hyperiongrabber/common/HyperionScreenService.java
@@ -161,8 +161,8 @@ public class HyperionScreenService extends Service {
     public int onStartCommand(Intent intent, int flags, int startId) {
         if (DEBUG) Log.v(TAG, "Start command received");
         super.onStartCommand(intent, flags, startId);
-        if (null == intent || null == intent.getAction()) {
-            String nullItem = (null == intent ? "intent" : "action");
+        if (intent == null || intent.getAction() == null) {
+            String nullItem = (intent == null ? "intent" : "action");
             if (DEBUG) Log.v(TAG, "Null " + nullItem + " provided to start command");
         } else  {
             final String action = intent.getAction();

--- a/common/src/main/java/com/abrenoch/hyperiongrabber/common/HyperionScreenService.java
+++ b/common/src/main/java/com/abrenoch/hyperiongrabber/common/HyperionScreenService.java
@@ -161,8 +161,11 @@ public class HyperionScreenService extends Service {
     public int onStartCommand(Intent intent, int flags, int startId) {
         if (DEBUG) Log.v(TAG, "Start command received");
         super.onStartCommand(intent, flags, startId);
-        final String action = intent.getAction();
-        if (action != null) {
+        if (null == intent || null == intent.getAction()) {
+            String nullItem = (null == intent ? "intent" : "action");
+            if (DEBUG) Log.v(TAG, "Null " + nullItem + " provided to start command");
+        } else  {
+            final String action = intent.getAction();
             if (DEBUG) Log.v(TAG, "Start command action: " + String.valueOf(action));
             switch (action) {
                 case ACTION_START:

